CONTRIBUTING.md

```diff
@@ -12,16 +12,18 @@ you will need to change this.
 
 Once you've gotten MySQL running you can run Lighthouse. It will take care of the rest.
 
-## Connecting
+## Connecting (PS3)
 
-PS3 is difficult to set up, so I will be going over how to set up RPCS3 instead. A guide will be coming for PS3 closer
-to release. You can also follow this guide if you want to learn how to modify your EBOOT.
+You can use [UnionPatcher's Remote Patch utility](https://github.com/LBPUnion/UnionPatcher) to manually patch your EBOOT.BIN; it works over the network and automatically downloads, decrypts, patches, encrypts, and uploads your PSN and Disc EBOOTs.
 
-Start by getting a copy of LittleBigPlanet 1/2 installed. (Check the LittleBigPlanet 1 section, since you'll need to do
+If you are using **RPCS3**, continue scrolling for the manual method.
```

Comment:
```suggestion
If you are not using a PS3, see [the RPCS3 section](#connecting-rpcs3)
```
I think this would flow better with the ps3 section if it said not ps3 at the end

Reply:
Done